utils/whatsapp/variables.go

```diff
@@ -0,0 +1,15 @@
+package whatsapp
```

Comment:
feels weird separating things into `structs.go` and `variables.go`... both of these could just go in `api.go`

Reply:
The go linter calls this "stuttering" tho only when it repeats exactly, e.g. `whatsapp.WhatsAppTemplate`

Reply:
ok